source/gui/guiHelper.py

```diff
@@ -273,4 +278,22 @@ def addLabeledControl(self, labelText, wxCtrlClass, **kwargs):
 		"""
 		labeledControl = LabeledControlHelper(self._parent, labelText, wxCtrlClass, **kwargs)
 		self.addItem(labeledControl.sizer)
-		return labeledControl.control
\ No newline at end of file
+		return labeledControl.control
+
+	def addDialogDismissButtons(self, buttons):
+		""" Adds and aligns the buttons for dismissing the dialog. These buttons may be "ok | cancel" and are expected

```

Comment:
It's perhaps not crystal clear here that OK and Cancel are just examples. Maybe it could say something like this:

`Adds and aligns the buttons for dismissing the dialog; e.g. OK and Cancel buttons. They are expected to be...`